.hooks/pre-commit.json

```diff
@@ -2,7 +2,7 @@
   "steps": [
     {
       "name": "check secrets",
-      "command": "npm run check-secrets"
+      "command": "[[ -n $CI ]] || npm run check-secrets"
```

Comment:
this is to prevent the git-commit that happens on version change from running secrets in ci?

Reply:
Yup! Didn't want to install git-secrets on ci every time